### PR TITLE
Compatibility with Symfony 3 and 4

### PR DESCRIPTION
**Pull request by ylcdx**
Change composer symfony/symfony to symfony/framework-bundle to make compatible with Symfony 3 and 4

### CODE REVIEW

**Comment by adelplace** [comment]
Hi,

Thanks for the contribution !
I need few days to do some tests on the newer versions then I'll get back to you.

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "Symfony integration of the norkunas/onesignal-php-api library",
     "type": "library",
     "require": {
-        "symfony/symfony": ">2.6",
+        "symfony/framework-bundle": "^3.3 || ^4.0",
```
Comment (adelplace) [inline composer.json]: Currently the norkunas/onesignal-php-api library isn't compatible with symfony 4 so you should remove the ^4.0 and keep the ^2.6 for retrocompatibility

**Comment by adelplace** [comment]
https://github.com/norkunas/onesignal-php-api/pull/92

**Comment by birkof** [comment]
@adelplace that PR has been accept. Give it a shot here too 🕺

**Comment by adelplace** [comment]
see https://github.com/adelplace/onesignal-bundle/pull/4

**Comment by birkof** [comment]
@adelplace, you should trigger packagist about these changes :D

**Comment by adelplace** [comment]
@birkof done ;)

**Pull request closed without merging**